src/pages/experiments/[experimentId]/plots-and-tables/PlotsTablesContainer.jsx

```diff
@@ -0,0 +1,169 @@
+
+import { plotNames, layout } from 'utils/constants';
+import heatmap from '../../../../../public/static/media/heatmap.png';
+import embeddingContinuous from '../../../../../public/static/media/embeddingContinuous.png';
+import embeddingCategorical from '../../../../../public/static/media/embeddingCategorical.png';
+import violin from '../../../../../public/static/media/violin.png';
+import dotPlot from '../../../../../public/static/media/dotplot.png';
+import volcano from '../../../../../public/static/media/volcano.png';
+import frequency from '../../../../../public/static/media/frequency.png';
+import markerHeatmap from '../../../../../public/static/media/marker_heatmap.png';
+
+const CARD_STYLE = { marginBottom: '1em' };
+const CardItem = (({ onClick, item, href }) => (
+  <Card.Grid
+    href={href}
+    onClick={onClick}
+    hoverable={false}
+    style={{ textAlign: 'center', width: '100%', padding: '0' }}
+  >
+    <img
+      alt={item.name}
+      src={item.image}
+      style={{
+        height: '200px', width: '250px', align: 'center', padding: '10px',
+      }}
+    />
+  </Card.Grid>
+));
+
+CardItem.defaultProps = {};
+
+CardItem.propTypes = {
+  item: PropTypes.object.isRequired,
+  href: PropTypes.string.isRequired,
+  onClick: PropTypes.func.isRequired,
+};
+
+const plots = [
+  {
+    title: 'Cell sets & metadata',
+    plots: [
+      {
+        name: plotNames.CATEGORICAL_EMBEDDING,
+        image: embeddingCategorical,
+        key: 'embedding-categorical-key',
+        link: 'embedding-categorical',
+      },
+      {
+        name: 'Frequency Plot',
+        image: frequency,
+        key: 'frequency-key',
+        link: 'frequency',
+      },
+    ],
+  },
+  {
+    title: 'Gene expression',
+    plots: [
+      {
+        name: plotNames.CONTINUOUS_EMBEDDING,
+        image: embeddingContinuous,
+        key: 'embedding-continuous-key',
+        link: 'embedding-continuous',
+      },
+      {
+        name: plotNames.MARKER_HEATMAP,
+        image: markerHeatmap,
+        key: 'marker-heatmap-key',
+        link: 'marker-heatmap',
+      },
+      {
+        name: plotNames.HEATMAP,
+        image: heatmap,
+        key: 'heatmap-key',
+        link: 'heatmap',
+      },
+      {
+        name: plotNames.VIOLIN_PLOT,
+        image: violin,
+        key: 'violin-key',
+        link: 'violin',
+      },
+      {
+        name: plotNames.DOT_PLOT,
+        image: dotPlot,
+        key: 'dot-key',
+        link: 'dot-plot',
+      },
+    ],
+  },
+  {
+    title: 'Differential expression',
+    plots: [
+      {
+        name: plotNames.VOLCANO_PLOT,
+        image: volcano,
+        key: 'volcano-key',
+        link: 'volcano',
+      },
+    ],
+  },
+];
+
+const PlotsTablesContainer = (props) => {
+  const { width, height, experimentId } = props;
+
+  return (
+    <div
+      style={{
+        width: width - (layout.PANEL_PADDING * 2),
+        height: height - (layout.PANEL_HEADING_HEIGHT * 2),
```

Comment:
I think the plot container should not have a heading at all. It just feels unnecessary to have an extra line saying "Plot". We were trying to remove stuff so I think that just having the "Reset" and "Info" buttons floating on the top right without a header would look much cleaner.

Reply:
We're trying to make the look more consistent throughout the platform. These additions make for a better UI and UX I think.